src/Bridge.ts

```diff
@@ -295,24 +319,23 @@ export class Bridge {
             await this.notifProcessor.onUserEvents(msg.data, adminRoom);
         });
 
-        this.queue.on<OAuthRequest>("oauth.response", async (msg) => {
-            const adminRoom = [...this.adminRooms.values()].find((r) => r.oauthState === msg.data.state);
+        this.queue.on<OAuthRequest>("github.oauth.response", async (msg) => {
+            const userId = this.tokenStore.getUserIdForOAuthState(msg.data.state);
```

Comment:
to prevent deletion which ends up half-bridging and account:
```suggestion
            const userId = this.tokenStore.getUserIdForOAuthState(msg.data.state, false);
```